### ISSUE TITLE
Can't read sdcard/nh_files/configs/hostapd-karma.conf: No such file or directory.

### ISSUE THREAD

**Issue by kepzbro** [offensive-security/nethunter-app#119]
Hello! I can't get mana toolkit or bad usb to work because of nethunter can't seem to find the directory or files in terminal. I have no problem looking into the folder with ÉS file explorer and all files seem to be there. Someone know how to fix this?

**Comment by binkybear** [comment]
Can you please provide more details such as what device/version of OS/version of software you are using.  Do you see files inside chroot in /sdcard?

**Comment by kepzbro** [comment]
Ok so this is what I have done so far: I upgraded my device version from lollipop 5.1.1 to marshmallow 6.0. then I reinstalled Nethunter 3.0 with the marshmallow rom instead. The install was successfull without any errors of what I could see. I started the phone went in terminal to get neccessary drivers for my Tp-link WN722N. Then I tried to get "ifconfig wlan1 up" now I get Error while getting interface flags: No such device. So for now I'm struggling with that.. I can't seem to find a "chroot" folder? in /sdcard. Using ES file explorer. I'm sorry I'm completly newb.. My device is a Nexus 6.
